src/views/webview-app/components/connect-helper/connect-helper.tsx

```diff
@@ -0,0 +1,102 @@
+import * as React from 'react';
+import classnames from 'classnames';
+import { connect } from 'react-redux';
+import {
+  ActionTypes,
+  LinkClickedAction,
+  OpenConnectionStringInputAction,
+  SetCurrentViewAction
+} from '../../store/actions';
+import { WEBVIEW_VIEWS } from '../../extension-app-message-constants';
+
+const styles = require('./connect-helper.less');
+
+type dispatchProps = {
```

Comment:
extremely silly question, isn't the convention for types to be like classes 'DispatchProps'?

Reply:
Ah it is hah. Updated ty.